Lib/doctest.py

```diff
@@ -1566,10 +1572,12 @@ def summarize(self, verbose=None):
         summary is.  If the verbosity is not specified, then the
         DocTestRunner's verbosity is used.
         """
+        from _colorize import ANSIColors, can_colorize
```

Comment:
I would also move these imports to the top of the module, since we already have `import _colorize` at the top (for the doctests).